packages/squidex/src/rest.ts

```diff
@@ -20,18 +20,25 @@ export interface Filter {
     | 'ne'
     | 'contains'
     | 'startsWith'
-    | 'endsWith';
-  value: string[] | number[] | string | number | boolean;
+    | 'endsWith'
+    | 'empty';
```

Comment:
This empty is a valid filter :) I've found out using Squidex UI and checking the network tab, but I also found it here https://github.com/Squidex/squidex/blob/6ac635a3b819145e0fec90ed724617b3819fe282/src/Squidex.Infrastructure/Queries/FilterOperator.cs#L13